src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/missingoverride/Example1.java

```diff
@@ -0,0 +1,42 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MissingOverride"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.annotation.missingoverride;
+
+// xdoc section -- start
+class Example1{
+  public void test1(){}
+
+  public void test2(){}
+
+}
+
+class A extends Example1 {
+
+  /** {@inheritDoc} */
+  @Override
+  public void test1() { // OK
+
+  }
+
+  /** {@inheritDoc} */
+  public void test2() { // violation, 'include @java.lang.Override'
+
```

Comment:
for other reviewers: I was not able to find better form of message that fits our limit on line lenght and reasonable wording. I think this form is most compact and ok.

Reply:
`// violation above, …`?

Reply:
Original message is very long, cut on its tail make it confusing, usage of .* also look weird.

Reply:
For documentation, readability and quick comprehension of examples is critical. If this is the best we can do with the existing constraints, we need to re-examine the constraints.